tests/executors/indexers/vector/test_annoy.py

```diff
@@ -56,6 +57,35 @@ def test_simple_annoy(self):
         _index.build(4)
         idx1, _ = _index.get_nns_by_vector(np.random.random((5,)), 3, include_distances=True)
 
+    def test_simple_ngt(self):
```

Comment:
You need to skip these tests if python3.8 is used.
```
import sys
PY37 = 'py37'
PY38 = 'py38'
py_tag = None
if sys.version_info >= (3, 8, 0):
    py_tag = PY38
elif sys.version_info >= (3, 7, 0):
    py_tag = PY37
else:
    raise OSError('Jina requires Python 3.7 and above, but yours is %s' % sys.version)

@unittest.skipIf('py38' == py_tag, 'skip if python3.8 version is tested')

```

Reply:
Much apppreciated !!